internal/core/worker/evolution/worker.go

```diff
@@ -68,7 +68,7 @@ func (w *worker) Run(ctx context.Context) error {
 				slog.Error(err.Error())
 				break
 			}
-            if lastBlock < startingBlock {
+			if lastBlock < startingBlock {
```

Comment:
can you remove these changes from this PR?

Reply:
As agreed in private, this change will stay in the PR